files/default/sql/ddl/2.1.0__initial_tables.sql

```diff
@@ -1516,6 +1530,9 @@ CREATE TABLE `training_dataset` (
   `training_dataset_type`   INT(11) NOT NULL DEFAULT '0',
   `seed` BIGINT(11) NULL,
   `query` TINYINT(1) NOT NULL DEFAULT '0',
+  `desc_stats_enabled` TINYINT(1) NOT NULL DEFAULT '1',
+  `feat_corr_enabled` TINYINT(1) NOT NULL DEFAULT '1',
+  `feat_hist_enabled` TINYINT(1) NOT NULL DEFAULT '1',
```

Comment:
Why are these here?

Reply:
thanks, that's a leftover, removed them.